WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -164,6 +224,38 @@ class PrepublishingViewController: UITableViewController {
         tableView.reloadData()
     }
 
+    private static func option(for identifier: PrepublishingIdentifier) -> PrepublishingOption {
+        switch identifier {
+        case .title:
+            return PrepublishingOption(id: .title, title: Constants.titlePlaceholder, type: .textField)
+        case .schedule:
+            return PrepublishingOption(id: .schedule, title: Constants.publishDateLabel, type: .value)
+        case .categories:
+            return PrepublishingOption(id: .categories, title: NSLocalizedString("Categories", comment: "Label for Categories"), type: .value)
+        case .visibility:
+            return PrepublishingOption(id: .visibility, title: NSLocalizedString("Visibility", comment: "Label for Visibility"), type: .value)
+        case .tags:
+            return PrepublishingOption(id: .tags, title: NSLocalizedString("Tags", comment: "Label for Tags"), type: .value)
+        }
+    }
+
+    private func setupTextFieldCell(_ cell: WPTextFieldTableViewCell) {
+        WPStyleGuide.configureTableViewTextCell(cell)
+        cell.delegate = self
+    }
+
+    // MARK: - Titls
+
+    private func configureTitleCell(_ cell: WPTextFieldTableViewCell) {
+        cell.textField.text = post.postTitle
+        cell.textField.textColor = .text
+        cell.textField.placeholder = "Title"
+        cell.textField.heightAnchor.constraint(equalToConstant: 40).isActive = true
+        cell.textField.autocorrectionType = .yes
+        cell.textField.autocapitalizationType = .sentences
+        cell.textField.becomeFirstResponder()
```

Comment:
The `becomeFirstResponder` makes this field to be selected every time the main screen appears, thus having this effect:


https://user-images.githubusercontent.com/7040243/108557855-e62a5080-72d7-11eb-8fce-200e2853d9c7.mov

I think it's a little bit confusing that the Bottom Sheet is moving around but I wouldn't consider that a block. Up to you to fix that or not!

Reply:
I had noticed this but wasn't sure the exact sequence. I've added a property `hasSelectedText` which causes `becomeFirstResponder` to only be called the first time the sheet is shown. I think this solves the issue. 👍